File: monitor/src/main/java/com/caibowen/prma/monitor/notify/Notifier.java
package com.caibowen.prma.monitor.notify;

import com.caibowen.prma.api.model.EventVO;

/**
 * @author BowenCai
 * @since 5-11-2014.
 */
public interface Notifier {

    void setName(String name);
    String getName();
    boolean notify(EventVO vo);
}

File: monitor/src/main/scala/com/caibowen/prma/monitor/notify/Notifier.scala
package com.caibowen.prma.monitor.notify


import com.caibowen.prma.monitor.eval.Response
import scala.beans.BeanProperty


/**
 * @author BowenCai
 * @since  01/12/2014.
 */
trait Notifier {

  @BeanProperty var name: String = _

  def take(vo: Response): Unit
}
